clang/lib/StaticAnalyzer/Core/CheckerManager.cpp

```diff
@@ -134,6 +135,14 @@ static void expandGraphWithCheckers(CHECK_CTX checkCtx,
 
 namespace {
 
+std::string checkerScopeName(StringRef name, const CheckerBase *checker) {
+  if (!llvm::timeTraceProfilerEnabled())
+    return "";
+  std::string checkerName =
```

Comment:
Nit: checkerName could be a StringRef removing one allocation.

Reply:
fixed
b5792c2b135e [NFC] Uppercase variable names